apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.IntegrationTests/ErrorReporting/ErrorEventEntryVerifiers.cs

```diff
@@ -14,13 +14,38 @@
 
 using Google.Cloud.ClientTesting;
 using Google.Cloud.ErrorReporting.V1Beta1;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
 using Xunit;
 
 namespace Google.Cloud.Diagnostics.Common.IntegrationTests
 {
-    public static class ErrorEventEntryVerifiers
+    internal static class ErrorEventEntryVerifiers
     {
+        /// <summary>
+        /// Checks that there's only one event present.
+        /// Includes a message with context info to help track the possible issue.
+        /// </summary>
+        public static ErrorEvent VerifySingle(ErrorEventEntryPolling polling, string contextInfo)
+        {
+            var events = polling.GetEvents(contextInfo, 1);
+            var errorEvent = events.SingleOrDefault();
+            Assert.True(errorEvent != null, $"No entries found for context {contextInfo}");
+            return errorEvent;
+        }
+
+        /// <summary>
+        /// Checks that there are <paramref name="expected"/> events present.
+        /// Includes a message with context info to help track the possible issue.
+        /// </summary>
+        public static IEnumerable<ErrorEvent> VerifyMany(ErrorEventEntryPolling polling, int expected, string contextInfo)
```

Comment:
Definitely not blocking a merge, but if you feel like reordering the parameters to put `expected` at the end, that would feel slightly better in terms of making `VerifySingle` equivalent to `VerifyMany(polling, contextInfo, 1)` except the return type. The thing that varies is just the number, so it's nice to put that last.

If this were production code I'd suggest actually doing it - as it is, it's mostly a drive-by stylistic comment :)

Reply:
Yes will change.